core/jvm/src/main/scala/scalaz/zio/lockfree/package.scala

```diff
@@ -0,0 +1,8 @@
+package scalaz.zio
+
+package object lockfree {
```

Comment:
Let's hide this in the queue since user-level code shouldn't have to call this explicitly.